radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -282,3 +282,36 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+// Author - Atharva (iya789)
+// May 7, 2022
+
+/**
+ *Resets microcontroller of the antenna
+ *
+ * @return 0 for success, non-zero for failure. See hal/errors.h for details.
+ */
+int antennaReset(void){
+
+	//command code for reset: 10101010 (section 6.2 of antenna user manual)
+
+	// form SSI library
+
+	int errorA = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideA );  //reset on side A
+	int errorB = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideB );  //reset on side B
+
+
+	if(errorA != 0) {
+
+			// TODO: record errors (if present) to System Manager
+			return errorA;
+		}
+
+	if(errorB != 0) {
+
+			// TODO: record errors (if present) to System Manager
+			return errorB;
+		}
```

Comment:
- indentation wrong in the body of the if blocks.
- I don't like the way this is done but I don't have a better suggestion. I think it's good for now. I would like to see what others say.

Reply:
I don't see the indentation fixed on the new commit. Double-check to make sure.